saleor/product/models.py

```diff
@@ -227,6 +231,14 @@ def __str__(self):
     def quantity_available(self):
         return max(self.quantity - self.quantity_allocated, 0)
 
+    @property
+    def is_visible(self):
+        return self.product.is_visible
+
+    @property
+    def is_available(self):
+        return self.product.is_in_stock()
```

Comment:
Shouldn't this use `product.is_available`?
```suggestion
        return self.product.is_available()
```

Reply:
Thanks!